File: Level-Order.cpp
#include <iostream>
#include <queue>

void DisplayLevelOrder(Node* root) {
    if (root == NULL)
        return;

    std::queue<Node*> q;
    q.push(root);

    while (!q.empty()) {
        Node* current = q.front();
        q.pop();
        std::cout << current->data << " ";

        if (current->left)
            q.push(current->left);
        if (current->right)
            q.push(current->right);
    }
}

File: Python/Level_Order.py
from queue import SimpleQueue

def DisplayLevelOrder(root):
    if root is None:
        return

    queue = SimpleQueue()
    queue.put(root)

    while not queue.empty():
        current = queue.get()
        print(current.data, end=" ")

        if current.left:
            queue.put(current.left)
        if current.right:
            queue.put(current.right)
